content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: blog
+title: "From Code to Quill: Embark on a Legendary Kubernetes Quest with SIG-Docs"
+date: 2024-04-28
+slug: contribute-as-sig-docs-reviewer
+author: >
+   Ricardo Amaro (Acquia)
+---
+
+You've probably come across the saying, "Contributing doesn't always mean writing lines of code"
+whispered to the wind or glimpsed in the margins of a sacred open-source tome.
```

Comment:
the [CNCF blog/style guide](https://www.cncf.io/blog/2018/09/04/the-cloud-native-computing-foundation-cncf-style-guide/) states:

> "Following this logic, “cloud native” and “open source” are not compound adjectives that need to be hyphenated. Thus, going forward, for all CNCF publications online and in print, we are establishing “open source” and “cloud native” as the proper style."

we recently added this to the Kubernetes style guide, but please go through the text and remove the hyphen so that its "open source" (i'm happy to go through and make suggested changes if you'd prefer that)

Reply:
fixing upstream also